llvm/lib/Target/AIE/AIEBaseSubtarget.cpp

```diff
@@ -584,6 +679,8 @@ AIEBaseSubtarget::getSMSMutationsImpl(const Triple &TT) {
   std::vector<std::unique_ptr<ScheduleDAGMutation>> Mutations;
   if (!TT.isAIE1()) {
     Mutations.emplace_back(std::make_unique<SWPWAWEdges>());
+    if (EnableWAWStickyRegisters)
+      Mutations.emplace_back(std::make_unique<WAWStickyRegistersEdges>());
```

Comment:
Would it make sense to enable that mutation in other schedulers?

Reply:
I observed no effects in other schedulers. Maybe we can keep it off and turn it on only if we see some opportunity in Post SWP (save time).